sw/device/lib/meson.build

```diff
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 subdir('base')
+subdir('testing')
```

Comment:
Oh, I've just read `index.md` and realized that putting tests under `sw/device/tests` was not a great idea! I should fix that in #1300.